primitives/version/src/lib.rs

```diff
@@ -71,6 +71,17 @@ pub struct RuntimeVersion {
 	/// will not attempt to author blocks unless this is equal to its native runtime.
 	pub authoring_version: u32,
 
+	/// All existing dispatches are fully compatible when this number doesn't change. If this
+	/// number changes, then `spec_version` must change, also.
+	///
+	/// This number must change when an existing dispatchable (module ID, dispatch ID) is changed,
+	/// either through an alteration in its user-level semantics, a parameter added/removed/changed,
+	/// a dispatchable being removed, a module being removed, or a dispatchable/module changing its
+	/// index.
+	///
+	/// It need *not* change when a new module is added or when a dispatchable is added.
+	pub transaction_version: u32,
```

Comment:
This needs to move to the bottom, or we brick everything^^

Reply:
Hmm that will not be enough. We need to change the version of the `Core` runtime api and make sure to put the `changed_in` attribute above the `version` function. 

The old `version` function should have the old `RuntimeVersion` struct as return value.

Reply:
Moved.